packages/ui/src/primitives/ErrorWell.tsx

```diff
@@ -27,6 +27,8 @@ export default function ErrorWell({ error, css }: Props) {
       txHash = (error as TransactionTimeoutError).txHash
     } else if (errorStatus >= 400 && errorStatus < 500) {
       message = error.message
+    } else if (error?.message?.includes('Crosschain')) {
+      message = error.message
```

Comment:
Don't really love the approach of checking the message. Can we maybe just throw an error with a type like we do for the price mismatch?